can-stache.js

```diff
@@ -424,7 +421,11 @@ function stache (filename, template) {
 			optionsScope.inlinePartials = optionsScope.inlinePartials || {};
 			assign( optionsScope.inlinePartials, inlinePartials );
 		}
+
 		scope.set('scope.view', scopifiedRenderer);
+		//!steal-remove-start
+		scope.set('scope.filename', section.filename);
+		//!steal-remove-end
 
 		// pass filename so it can be used in dev warnings
 		optionsScope._meta.filename = section.filename;
```

Comment:
do we still need the filename added to `optionsScope._meta`?

Reply:
I believe it is not.